README.md

```diff
@@ -23,6 +23,7 @@ Creating:
   event.SetLocation("Address")
   event.SetDescription("Description")
   event.SetURL("https://URL/")
+  event.AddRrule(fmt.Sprintf("FREQ=YEARLY;BYMONTH=%d;BYMONTHDAY=%d", time.Now().Month(), time.Now().Day()))
```

Comment:
I will have to figure out better way of communicating that info. That's a bit cumbersome. But good for now.